src/vs/workbench/contrib/terminal/browser/terminalInstance.ts

```diff
@@ -518,13 +518,6 @@ export class TerminalInstance extends Disposable implements ITerminalInstance {
 			throw new Error('xterm elements not set after open');
 		}
 
-		// Check if custom Terminal title exists and set same
-		if (this._title.length > 0) {
-			xterm.textarea.setAttribute('aria-label', nls.localize('terminalTextBoxAriaLabelNumberAndTitle', "Terminal {0}, {1}", this._id, this._title));
-		} else {
-			xterm.textarea.setAttribute('aria-label', nls.localize('terminalTextBoxAriaLabel', "Terminal {0}", this._id));
-		}
-
```

Comment:
Thanks for the fix! I think we also need to update the title here for the following case:

1. Terminal launches
2. Terminal title gets set (textarea didn't exist)
3. Attach is called

Can the code to update `aria-label` be factored out into a new function and called here also?

Reply:
I have put it in a private function and called it in `_attachToElement`. Is there anything left to do, to fulfill 1-3?